src/ping/ping.js

```diff
@@ -52,7 +52,7 @@ module.exports = (createCommon, options) => {
       })
     })
 
-    it('should fail when pinging a peer that is not available', (done) => {
+    it.skip('should fail when pinging a peer that is not available: go-ipfs returns success', (done) => {
```

Comment:
Needs to be skipped in `ipfs-http-client` https://github.com/ipfs/js-ipfs-http-client/blob/master/test/interface.spec.js#L237